### ISSUE TITLE
Disabled password Entry should also disable the ActionItem (show/hide password)

### ISSUE THREAD

**Issue by matwachich** [fyne-io/fyne#3908]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

When an Entry with .Password=true is disabled, the Action item button should also be disabled.

### How to reproduce

Disable a Password Entry

### Example code

```go
package main

func main() {
    ...

    entry := widget.NewEntry()
    entry.Password = true
    entry.Disable()

    ...
}
```

### Fyne version

develop v2.3.4-rc1.0.20230516195251-56dd0f3e0f32

### Go compiler version

1.20

### Operating system and version

Windows 10

**Comment by Jacalz** [comment]
Please write a fully run-able example code next time. I know that it might seem unnecessary and time consuming sometimes but it really is very helpful both when adding test cases and when testing pull requests locally.

**Comment by andydotxyz** [comment]
Fixed on develop and release/v2.3.x branch ready for v2.3.5
